exonum-light-client/src/test/java/com/exonum/client/ExplorerApiHelperTest.java

```diff
@@ -232,5 +234,8 @@ void parseServicesResponse() {
 
     List<ServiceInfo> actual = ExplorerApiHelper.parseServicesResponse(json);
     assertThat(actual, contains(expected.toArray()));
+    int serviceId = Optional.of(serviceInfo1)
+        .map(ServiceInfo::getId)
```

Comment:
Will `mapToInt` be more appropriate (in the example also)? OptionalInt also has https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/OptionalInt.html#orElseThrow(java.util.function.Supplier)

Reply:
Sorry, either I don't understand what exactly do you propose here or it's something like
```
    int serviceId = Optional.of(serviceInfo1).stream()
        .mapToInt(ServiceInfo::getId)
        .findFirst()
        .orElseThrow(() -> new IllegalStateException("No service with the given name found: " + serviceName);
```
which doesn't seem better to me.
`Optional` doesn't have `mapToInt` method.

Reply:
I see, sorry, I must have missed there is no mapToInt in Optional. It's good then, a stream is certainly inferior.